src/platform/site-wide/announcements/tests/components/SingleSignOnInfoModal.unit.spec.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { expect } from 'chai';
+import { mount } from 'enzyme';
+import sinon from 'sinon';
+
+import localStorage from 'platform/utilities/storage/localStorage';
+import SingleSignOnInfoModal from '../../components/SingleSignOnInfoModal';
+
+describe('Announcements <SingleSignOnInfoModal>', () => {
+  it('renders for logged in SSO users', () => {
+    localStorage.setItem('hasSessionSSO', true);
+    const dismiss = sinon.spy();
+
+    const tree = mount(<SingleSignOnInfoModal isLoggedIn dismiss={dismiss} />);
+
+    expect(tree.text()).to.contain(
+      'Sign in once to access the VA sites you use most',
+    );
+
+    tree
+      .find('button')
+      .at(1)
+      .props()
+      .onClick();
+    expect(dismiss.called).to.be.true;
```

Comment:
Seems like this is doing more than the test case describes. It'd be clearer to create a separate one to address the dismiss handler.

Reply:
I based that test off of the other announcement modal tests, but I see what you mean. I split it out into its own unit test.